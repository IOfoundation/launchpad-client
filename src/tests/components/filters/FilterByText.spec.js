// TODO: test deleteFilter, clearAll, handleDropdownOnClick (should trigger handleOnChangeFilterOptions()), test renderFilter (should return filter chips),
//       test _inputClick should change state of labelTop, test handleKeyPress & renderDropdown (should return with a dropdown of results)
