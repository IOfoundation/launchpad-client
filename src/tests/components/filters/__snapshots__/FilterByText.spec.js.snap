// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FilterByText /> renders snapshot of FilterByOptions 1`] = `
<FilterByText
  appliedFilters={
    Object {
      "page": 1,
    }
  }
  disableOnClickOutside={[Function]}
  enableOnClickOutside={[Function]}
  eventTypes={
    Array [
      "mousedown",
      "touchstart",
    ]
  }
  filterById={false}
  filterName="Search Box"
  getTextSearchResults={[Function]}
  handleClickOnClearAllFilters={[Function]}
  handleOnChangeFilterOptions={[Function]}
  items={Array []}
  organizations={
    Array [
      Object {
        "contacts": Array [
          Object {
            "email": "jondoe@test.org",
            "id": 1,
            "name": "Jon Doe",
            "phones": Array [
              Object {
                "number": "(530) 795-1520",
              },
            ],
          },
        ],
        "description": "orgDescription",
        "email": "jondoe@something.org",
        "id": 1,
        "locations": Array [
          Object {
            "address": Object {
              "address_1": "123 Road St",
              "address_2": null,
              "city": "Sacramento",
              "postal_code": "12345",
              "state_province": "CA",
            },
            "id": 1,
            "name": "location1",
            "phones": Array [
              Object {
                "id": 2,
                "number": "(123) 456 - 7890",
              },
            ],
          },
          Object {
            "address": Object {
              "address_1": "456 Street Rd",
              "address_2": null,
              "city": "Sacramento",
              "postal_code": "12345",
              "state_province": "CA",
            },
            "id": 2,
            "name": "location2",
            "phones": Array [
              Object {
                "id": 3,
                "number": "(123) 456 - 7890",
              },
            ],
          },
        ],
        "name": "organization1",
        "phones": Array [
          Object {
            "id": 1,
            "number": "(123) 456 - 7890",
          },
        ],
        "services": Array [
          Object {
            "categories": Array [
              Object {
                "id": 54,
                "name": "category1",
              },
              Object {
                "id": 55,
                "name": "category2",
              },
              Object {
                "id": 56,
                "name": "category3",
              },
            ],
            "description": "lorem ipsum...",
            "email": null,
            "id": 1,
            "name": "service1",
            "phones": Array [],
          },
          Object {
            "categories": Array [
              Object {
                "id": 54,
                "name": "category1",
              },
            ],
            "description": "lorem ipsum...",
            "email": null,
            "id": 2,
            "name": "service2",
            "phones": Array [],
          },
        ],
      },
    ]
  }
  outsideClickIgnoreClass="ignore-react-onclickoutside"
  preventDefault={false}
  stopPropagation={false}
/>
`;
